lib/services/login.dart

```diff
@@ -0,0 +1,32 @@
+import 'dart:async';
+import 'dart:convert';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'post.dart';
+import '../model/user.dart';
+
+class Login {
+  static final baseUrl = "https://apistaginghr.excellencetechnologies.in";
+  static final loginUrl = baseUrl + "/attendance/API_HR/api.php";
```

Comment:
these should be moved to configuration or env file. there should be something for dart

Reply:
ok creating app config file